website/content/docs/connect/config-entries/jwt-provider.mdx

```diff
@@ -436,6 +511,79 @@ Specifies a jittered exponential backoff strategy. When this field is empty, Env
+  - [`DiscoveryType`](#jsonwebkeyset-remote-jwkscluster-discoverytype)
+  - [`ConnectTimeout`](#jsonwebkeyset-remote-jwkscluster-connecttimeout)
+
+### `JSONWebKeySet{}.Remote{}.JWKSCluster{}.ConnectTimeout`
+
+Specifies the timeout for new network connections to hosts in the cluster.
+
+#### Values
+
+- Default: `5s`
+- Data type: String
+
+### `JSONWebKeySet{}.Remote{}.JWKSCluster{}.DiscoveryType`
+
+Specifies the service discovery type to use for resolving the cluster.
+The discovery type options are: STRICT_DNS, STATIC, LOGICAL_DNS, EDS and ORIGINAL_DST.
+
+#### Values
+
+- Default: STRICT_DNS
+- Data type: String
+
+### `JSONWebKeySet{}.Remote{}.JWKSCluster{}.TLSCertificates`
+
+Specifies the data containing certificate authority certificates to use in
+verifying a presented peer certificate. If not specified and a peer certificate is presented
+it will not be verified. You cannot specify [`TLSCertificates{}.CaCertificateProviderInstance`](#jsonwebkeyset-remote-jwkscluster-tlscertificates-cacertificateproviderinstance) and [`TLSCertificates{}.TrustedCA`](#jsonwebkeyset-remote-jwkscluster-tlscertificates-trustedca) in the same map.
+
+#### Values
+
+- Default: None
+- Data type: Map that can contain the following parameters:
+
+  - [`CaCertificateProviderInstance`](#jsonwebkeyset-remote-jwkscluster-tlscertificates-cacertificateproviderinstance)
+  - [`TrustedCA`](#jsonwebkeyset-remote-jwkscluster-tlscertificates-trustedca)
+
+### `JSONWebKeySet{}.Remote{}.JWKSCluster{}.TLSCertificates{}.CaCertificateProviderInstance`
+
+Speficies the certificate provider instance for fetching TLS certificates.
+
+#### Values
+
+- Default: None
+- Data type: Map that can contain the following parameters:
+
+| Parameter | Description                                        | Data type | Default value |
+| :-------- | :------------------------------------------------- | :-------- | :------------ |
+| `InstanceName`| Refers to the certificate provider instance name. | String | `default` |
+| `CertificateName` | certificate instances or types. For example, "ROOTCA" to specify a root-certificate. | String | None |
+ 
+### `JSONWebKeySet{}.Remote{}.JWKSCluster{}.TLSCertificates{}.TrustedCA`
+
+Specifies TLS certificate data containing certificate authority certificates. Exactly one of Filename, EnvironmentVariable, InlineString or InlineBytes must be specified.
+
+#### Values
+
+| Parameter | Description                                        | Data type | Default value |
+| :-------- | :------------------------------------------------- | :-------- | :------------ |
+| `Filename`| Local filesystem data source.                      | String    | None          |
+| `EnvironmentVariable` | Environment variable data source.      | String    | None          |
+| `InlineString` | String inlined in the configuration.          | String    | None          |
+| `InlineBytes` | Bytes inlined in the configuration.            | String    | None          |
```

Comment:
```suggestion
| `InlineString` | String inlined in the configuration.          | String    | None          |
| `InlineBytes` | Bytes inlined in the configuration.            | String    | None          |
```
Would it be obvious to our audience what we mean by "inlined"? I was trying come up with alternate language to propose, but I don't understand what this means.

Reply:
yeah picked it directly from the envoy docs: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/base.proto.html#envoy-v3-api-field-config-core-v3-datasource-inline-bytes